services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Perustaja.kt

```diff
@@ -0,0 +1,7 @@
+package fi.hel.haitaton.hanke.domain
+
+import fi.hel.haitaton.hanke.PerustajaEntity
+
+data class Perustaja(val nimi: String?, val sahkoposti: String)
```

Comment:
Should sahkoposti be named email instead, so it would be consistent, as email is used elsewhere, for example in HankeYhteystieto?

Reply:
This implementation is following the specification mentioned in: https://helsinkisolutionoffice.atlassian.net/browse/HAI-1349

But it is good to be consistent so I will change this as suggested.

Reply:
Done.